### ISSUE TITLE
"Identified by" overwritten if name not in list (+ save error)

### ISSUE THREAD

**Issue by cjcampi** [paleocore/paleocore#165]
Need to create a "people" table ASAP.

For a record where the "Identified by" is NOT currently a dropdown option, it will show "----" on webpage, but DOES have/show name in .csv export (Access database info).
However, if you try to make any edits to that record in any field, it 1) officially overwrites the db and assigns "----" for "identified by (so old information now lost) and since that is part of the "Taxonomy" module, the other edits are not saved and have to be re-done (see Issue 163). However, there is no way to know this is the case unless you re-check the record. I only noticed this because when a record didn't save, I looked at the history and it said I edited "identified by", which I most definitely didn't.

**Comment by dennereed** [comment]
Chris, can you provide an example record where this happens?

**Comment by cjcampi** [comment]
Check out barcode 114321.
My previous csv export had J.A. Harris as "identified by" (but was ----- on webpage). My new csv export has that field as blank. Also check the history. You'll see how it says I edited "identified by" plus the other fields, but in fact PaleoCore edited the "identified by" and did not save my actual edits, which I then had to re-do. 
If you want to test, try editing barcode 113545 which also has J.A. Harris as "identified by" in my updated csv export. Write "test" in "Collection Remarks" or something then try to save it. It will say that you updated that field and identified by, but it won't update "Collection Remarks", and J.A. Harris will be removed from "identified by" in the next csv export.

**Comment by dennereed** [comment]
How did J.A. Harris get into the database?  The database you sent me that I uploaded into paleocore did not include J.A. Harris as a collector, just J. Harris.
Distinct entries for Collector....
(null)
A.E. Shapiro
B. Villamoare
B.J. Schoville
C. Seyoum
C.J. Campisano
D. Feary
D. Garello
D. I. Garello
D.N. Reed
D.R. Braun
E. Locke
E. Scott
E.N. DiMaggio
HFS Student
HRP Team
I. Lazagabaster
I. Smail
J. Harris
J. Robinson
J. Rowan
K.E. Reed
L. Verdelin
L. Werdelin
M. Bamford
S. Oestmo
T.K. Nalley
Tomas
W.H. Kimbel
Zinash

**Comment by cjcampi** [comment]
Dunno, this is what happens when you let graduate students and postdocs enter data while geologists are out doing real work. It's possible one of them did it so that it wasn't confused with John M. (J.M. Harris) who identified a bunch of the Hadar stuff back in the day.

**Comment by dennereed** [comment]
Add a function to lgrp.models.Biology that validates the field values against the choice lists in lgrp.ontologies. Ran this for every field that has choices.  Where unmatched choices were found I updated the DB to match an existing choice, OR I added the choice to lgrp.ontologies.  Two fields, 'finder' and 'element' were most problematic and the values in those fields needs to be reviewed and updated.  The problematic entries are listed in lgrp.ontologies as LGRP_FINDER_CHOICES and LGRP_ELEMENT_ADDITIONAL.
